.pre-commit-config.yaml

```
@@ -1,4 +1,9 @@
 repos:
+  -   repo: https://github.com/pycqa/isort
+      rev: 5.7.0
```

It looks like release came out a few days ago

```suggestion
      rev: 5.8.0
```